spec/requests/users_controller_spec.rb

```diff
@@ -123,6 +130,10 @@
           expect(page).not_to have_link("Deactivate user", href: "/users/#{user.id}/deactivate")
         end
 
+        it "does not allow resending invitation emails" do
+          expect(page).not_to have_link("Resend invite link", href: "/users/#{other_user.id}/resend-invite")
```

Comment:
this is great. compare to line 82 where someone did something 💩  instead.

however, again change to _path helper please.

Reply:
Oh 82 is fun yeah. 
All paths changed I to path helper in upcoming commit